src/server/game/Miscellaneous/Language.h

```diff
@@ -811,6 +811,9 @@ enum TrinityStrings
     LANG_PINFO_CHR_PLAYEDTIME           = 853,
     LANG_PINFO_CHR_MAILS                = 854,
     LANG_PINFO_CHR_LEVEL_HIGH           = 871,
+    LANG_NO_REASON             			= 35407,
+    LANG_UNAUTHORIZED             		= 35408,
```

Comment:
And this

Reply:
I want to edit but It's fine no have space !